src/briefcase/commands/create.py

```diff
@@ -618,7 +639,7 @@ def _install_app_requirements(
             self.tools.os.mkdir(app_packages_path)
 
         # Install requirements
-        if requires:
+        if requires or app.requirement_installer_args:
```

Comment:
I'm not sure I understand this one - why invoke `pip install`  if there are pip arguments, but aren't any requirements to install?

Reply:
A user could define a requirement in the new option, for example, to install it as editable. Maybe the "right" way to do that is `--editable=../package` in `requires`. Regardless, it's an edge case that (probably) only applies in a development environment, and using `requires` for it would still work. There may be other such examples, I'm not sure. I don't have a strong feeling about this, happy to revert it.

Reply:
I think `--editable` doesn't really make sense in a Briefcase context, much like `--user` and other options which change the location of the installed code. Briefcase's equivalent to `--editable` is `briefcase dev`.

Reply:
Wouldn't you want to use editable in, say, a monorepo situation, so you don't have to re-`pip install` (via `briefcase dev -r`)? I've done this myself a bit when working on Toga GTK features that I wanted to try out in an actual application, it makes it much nicer than needing to wait for `pip install` each time I restart `briefcase dev`.

Maybe I've missed some part of the dev workflow there, though.

Reply:
You're correct that editable installs can be useful for `briefcase dev` purposes - I do the same thing as you've described for Toga development. Editable installs *might* even work in some scenarios for a `briefcase run` purposes - but they definitely won't on anything deployment target that is sandboxed (e.g., Flatpak, or anything mobile) because the `.pth` file that manifests the editable install won't resolve at runtime. 

In terms of this specific feature, I could probably go either way. An empty `requires` is definitely a niche edge case; I think we can leave it as right now, and if it turns out to be necessary or useful, it's an easy addition later on.

Reply:
Sounds good! Thanks for the input, all. I'll resolve this thread as addressed in https://github.com/beeware/briefcase/pull/2059/commits/b67eed533818c10ee48da41b3e7b8de712dabfe8.